src/components/Icons.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+interface IconProps {
+  className?: string
+}
+
+export const CorrectAnswerIcon: React.FC<IconProps> = ({ className }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    className={className}
+  >
+    <path
+      fill="#019920"
+      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10s10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5l1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
+    />
+  </svg>
+)
+
+export const WrongAnswerIcon: React.FC<IconProps> = ({ className }) => (
```

Comment:
Super minor: This declaration is somewhat inconsistent in this code base.

Reply:
Updated per this [commit](https://github.com/openstax/k12-apps-raise/pull/89/commits/040f8931d9c2d6451b0d23a1920bccde6a4843cd).